packages/components/tests/dummy/app/styles/pages/db-gutters.scss

```diff
@@ -0,0 +1,47 @@
+// GUTTERS
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+    --hds-spacing-xxs: 4px;
+    --hds-spacing-xs: 8px;
+    --hds-spacing-sm: 12px;
+    --hds-spacing-md: 16px;
+    --hds-spacing-lg: 24px;
+    --hds-spacing-xl: 36px;
+    --hds-spacing-xxl: 48px;
+}
+
+// TODO: Determine best default spacing size
+$default-spacing: var(--hds-spacing-md); // set as Sass variable to not expose
+
+.hds-gutters {
+    // vertical (make default ?): (Find out most common use cases)
+    // &:not([class*="hds-gutters-horizontal"]), // yes/ no?
```

Comment:
I wasn't sure if "direction" should be a required argument or if a default should be set.

Reply:
let's avoid to use `:not` pseudo selectors, they created havoc when used before :)